src/streambench/sparkbench/src/main/scala/com/intel/hibench/streambench/spark/RunBench.scala

```diff
@@ -23,6 +23,7 @@ import com.intel.hibench.streambench.spark.microbench._
 
 object RunBench {
   var reportDir = ""
+  var counts = 0L;
```

Comment:
I think we don't need this. We can use `params.recordCount` in `LatencyCollector`